From fa7a1ef6bc1fe26d871bf9d1dfc74d45a7b010a2 Mon Sep 17 00:00:00 2001
From: Michael Anderson <anderson@mbari.org>
Date: Thu, 22 Sep 2022 22:35:56 -0700
Subject: [PATCH 1/1] fix utilities include

Signed-off-by: Michael Anderson <anderson@mbari.org>
---
 include/utilities.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/utilities.h b/include/utilities.h
index ebeb68a..f2f603e 100644
--- a/include/utilities.h
+++ b/include/utilities.h
@@ -12,7 +12,7 @@
 
 #include <vector>
 #include <stdlib.h> // std::abs etc
-#include <definitions.h>
+#include "definitions.h"
 
 namespace SPLINTER
 {
-- 
2.37.3

